fundamentals/middleware/problem-details-service/Program.cs

```diff
@@ -70,16 +77,81 @@
 });
 
 app.Run();
+// </snippet_1>
+#elif MIDDLEWARE
```

Comment:
@sammychinedu2ky  use this `elif` for the middleware.

Reply:
alright would do that